core.py

```diff
@@ -134,7 +134,20 @@ def cli():
 )
 @click.option("--whodrug", help="Path to directory with WHODrug dictionary files")
 @click.option("--meddra", help="Path to directory with MedDRA dictionary files")
-@click.option("--rules", "-r", multiple=True)
+@click.option(
+    "--rules",
+    "-r",
+    multiple=True,
+    help="specify rule core ID ex. CORE-000001. Can be specified multiple times",
+)
+@click.option(
+    "--unpublished",
+    "-u",
+    required=False,
+    multiple=True,
+    type=click.Path(exists=True, readable=True, resolve_path=True),
+    help="Absolute path to unpublished rule. Can be specified multiple times",
```

Comment:
Is this a directory or file? I think a directory would be more convenient than a file specified multiple times.
If necessary, it can still be used in combination with -r to filter out the specific rules to be targeted.